README.txt

Golutazem opened the pull request:
Elaborate license text. Remove specific speed statement.

Commit:
Remove mention of 10m/s limit, since that is not used.

Commit:
Elaborate public domain non-copyright.

SmallJoker commented on README.txt:
```diff
@@ -7,17 +7,19 @@ Target: Run smoothly as possible even on laggy server
-CC-0, if not specified otherwise below
+Except for specific works which have other conditions listed in the Authors section,
+creators reserve no copyright or other rights to this work.
+This work, with specific exception of some parts, is in the public domain.
```
Public domain can vary for each country, thus this should stay CC0.

SmallJoker commented on README.txt:
```diff
@@ -7,17 +7,19 @@ Target: Run smoothly as possible even on laggy server
 
- License for everything
+ Copying / Copyright
 ------------------------
```
Could you please also adjust the amount of dashes here? (length of title + 2 in total)

SmallJoker reviewed (changes requested):
See comments

Commit:
Make node references absolute. Accomodate different top directory name.

Commit:
Remove statement about public domain.

Golutazem commented:
New commits.

Removed statement about public domain. If one has given up rights , then granting a license seems a little self-contradictory -- because if one has no exclusive rights, one has no right to grant.  CC0 was also not explained. For that reason, and that the CC0 WWW site stinks (in my Firefox), I've removed CC0.

New in the pull request is ability for the module to operate in any directory. One must only set a name.
